index.html

```diff
@@ -23,14 +23,25 @@ <h1>👋 I'm Gabe</h1>
         Ideally these spans would be buttons to get all the functionality for free, but buttons
         cannot be true inline elements and therefore cannot wrap like normal text. So instead we
         use spans and have manually added in the functionality to make it accessible. -->
-      <!-- TODO, follow this guide: https://benfrain.com/converting-divs-into-accessible-pseudo-buttons/ -->
       <p>
-        I’m a Senior Software Engineer who enjoys <span>fostering inclusive teams</span>,
-        <span>building robust codebases</span>, and <span>tackling hard technical challenges</span>.
+        I’m a Senior Software Engineer who enjoys <span role="button" aria-pressed="false" tabindex="0"
+          class="vibe-button">fostering
+          inclusive teams</span>,
+        <span role="button" aria-pressed="false" tabindex="0" class="vibe-button">building robust
+          codebases</span>, and
+        <span role="button" aria-pressed="false" tabindex="0" class="vibe-button">tackling hard
+          technical
+          challenges</span>.
       </p>
       <p>
-        Outside of work (but sometimes at work as well 🙂) I like to <span>cook</span>, <span>bake</span>,
-        <span>play video games with friends</span>, and <span>vibe with my cat
+        Outside of work (but sometimes at work as well 🙂) I like to <span role="button" aria-pressed="false"
```

Comment:
Optional suggestion: I think it would be a neat exercise to implement this as a `<span-button>` custom element to reduce boilerplate/duplication.

https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements

Reply:
Great suggestion! Done!